docs/documentation/_data/sidebars/main.yml

```diff
@@ -1101,6 +1101,29 @@ entries:
                 en: Configuration
                 ru: Настройки
               url: /modules/450-network-gateway/configuration.html
+        - title: virtualization
+          featureStatus: experimental
+          d8Revision: ce
+          folders:
+            - title:
+                en: Description
+                ru: Описание
+              url: /modules/490-virtualization/
+            - title:
+                en: Usage
+                ru: Примеры
+              url: /modules/490-virtualization/usage.html
```

Comment:
rename to examples.html

Reply:
done

Reply:
updated in 3368733bc5f6bb255767db9283876b1756b46359